README.md

```diff
@@ -38,22 +40,72 @@ The closing brace for `guard` statements should be followed by an empty line.
 
 ---
 
+##### An empty line should follow an opening brace or start of a closure
+
+*Preferred*
+```swift
+override func viewDidLoad() {
+
+    // some code here. Notice the empty line above this one.
+}
+
+// a method with a closure parameter
+myMethod() {
```

Comment:
I believe elsewhere in the style guide, we specify these parentheses should be omitted, so we should fix that.  (If we don't specify it, we should...)  We can just add an argument inside the parenthesis.